lib/src/utilities/extract-endpoint-types.ts

```diff
@@ -2,15 +2,10 @@ import { compact } from "lodash";
 import { Locatable } from "../models/locatable";
 import { EndpointNode, TypeNode } from "../models/nodes";
 
-export function extractEndpointTypes(
+export function extractResponseTypes(
```

Comment:
It looks like both `extractRequestTypes` and `extractResponseTypes` return an array with at most one element. How about changing their return type to `TypeNode | null` instead?

Reply:
`extractResponseTypes` can return an array with more than 1 element -> `endpoint.value.responses.map(` and default response.

Reply:
Good point. What about `extractRequestTypes`?